portfolio/pom.xml

```diff
@@ -16,7 +16,7 @@
 
     <!-- Project-specific properties -->
     <exec.mainClass>com.google.sps.ServerMain</exec.mainClass>
-    <googleCloudProjectId>YOUR_PROJECT_ID_HERE</googleCloudProjectId>
+    <googleCloudProjectId>elite-hangar-315818</googleCloudProjectId>
```

Comment:
Out of curiosity where did you get this project ID? I would have expected something like `kdiaz-sps-summer21` which should have been generated for you in your SPS account already?

Reply:
Hi, I realized after I had to use my SPS account instead of my normal Google account. I fixed it with the latest commit and pull request.

Reply:
Awesome, that makes sense!